diff --git a/dist/index.js b/dist/index.js
index 18cf08f5014a4385f256005b02d4d23687a68cfb..cb610f501f53a730bad4483961e9f531d16848eb 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -882,7 +882,7 @@ async function node(options = {}) {
 			"node/no-exports-assign": "error",
 			"node/no-new-require": "error",
 			"node/no-path-concat": "error",
-			"node/no-sync": "error",
+			"node/no-sync": "off",
 			"node/prefer-global/buffer": ["error", "never"],
 			"node/prefer-global/console": ["error", "always"],
 			"node/prefer-global/url": ["error", "always"],
diff --git a/src/config/node.ts b/src/config/node.ts
index 6156c8d0d65c42a2cb320187f0bba1fded226dd1..80364c8caef08fcd7e8d2ab702dc1f5a3129099b 100644
--- a/src/config/node.ts
+++ b/src/config/node.ts
@@ -22,7 +22,7 @@ export async function node(options: node.Options = {}) {
         'node/no-exports-assign': 'error',
         'node/no-new-require': 'error',
         'node/no-path-concat': 'error',
-        'node/no-sync': 'error',
+        'node/no-sync': 'off',
         'node/prefer-global/buffer': ['error', 'never'],
         'node/prefer-global/console': ['error', 'always'],
         // 'node/prefer-global/process': ['error', 'never'],
